spec/requests/objects_controller_file_spec.rb

```diff
@@ -10,46 +10,48 @@
     context 'when object exists' do
       context 'when druid is prefixed' do
         it 'returns the requested file' do
-          get file_object_url(id: prefixed_druid), params: { category: 'manifest', filepath: 'manifestInventory.xml' }
+          get file_object_url(id: prefixed_druid), params: { category: 'manifest', filepath: 'manifestInventory.xml' }, headers: valid_auth_header
           expect(response).to have_http_status(:ok)
           expect(response.body).to include 'md5'
         end
       end
 
       context 'when druid is bare' do
         it 'returns the requested file' do
-          get file_object_url(id: bare_druid), params: { category: 'manifest', filepath: 'manifestInventory.xml' }
+          get file_object_url(id: bare_druid), params: { category: 'manifest', filepath: 'manifestInventory.xml' }, headers: valid_auth_header
           expect(response).to have_http_status(:ok)
           expect(response.body).to include 'md5'
         end
       end
 
       context 'when file is not present in Moab' do
         it 'returns 404 response code; body has additional information' do
-          get file_object_url(id: prefixed_druid), params: { category: 'manifest', filepath: 'foobar' }
+          get file_object_url(id: prefixed_druid), params: { category: 'manifest', filepath: 'foobar' }, headers: valid_auth_header
           expect(response).to have_http_status(:not_found)
           expect(response.body).to eq '404 Not Found: manifest file foobar not found for bj102hs9687 - 3'
         end
       end
 
       context 'when version specified' do
         it 'retrieves the correct version of the file' do
-          get file_object_url(id: prefixed_druid), params: { category: 'manifest', filepath: 'manifestInventory.xml', version: '3' }
+          params = { category: 'manifest', filepath: 'manifestInventory.xml', version: '3' }
```

Comment:
don't really care but ... why is this one not like the others (one line)

Reply:
line length again, i believe